src/main/java/org/jabref/gui/maintable/CellFactory.java

```diff
@@ -10,96 +10,103 @@
 import org.jabref.gui.externalfiletype.ExternalFileType;
 import org.jabref.gui.externalfiletype.ExternalFileTypes;
 import org.jabref.gui.icon.IconTheme;
+import org.jabref.gui.icon.JabRefIcon;
 import org.jabref.gui.specialfields.SpecialFieldViewModel;
 import org.jabref.model.entry.FieldName;
 import org.jabref.model.entry.specialfields.SpecialField;
+import org.jabref.model.strings.StringUtil;
 
 public class CellFactory {
 
-    private final Map<String, Node> TABLE_ICONS = new HashMap<>();
+    private final Map<String, JabRefIcon> TABLE_ICONS = new HashMap<>();
 
     public CellFactory(ExternalFileTypes externalFileTypes, UndoManager undoManager) {
-        Node label;
-        label = IconTheme.JabRefIcons.PDF_FILE.getGraphicNode();
-        //label.setToo(Localization.lang("Open") + " PDF");
-        TABLE_ICONS.put(FieldName.PDF, label);
+        JabRefIcon icon;
+        icon = IconTheme.JabRefIcons.PDF_FILE;
+        //icon.setToo(Localization.lang("Open") + " PDF");
```

Comment:
What about the tooltips @tobiasdiez ? I think we should have them

Reply:
Tooltips should indeed be reimplemented. However, I don't think that these old generic ones are really helpful. Instead of "Open PDF" I would prefer a tooltip that really explains what will happen if you click on the icon, i.e "Open filename.pdf".

Not sure if this should be part of this PR or a follow-up.

Reply:
Lets do it as a follow up?